src/utils/CredentialUtils.ts

```diff
@@ -46,10 +46,10 @@ export const getOriginalVerifiableCredential = (vc: VerifiableCredential | ICred
   return CredentialMapper.toWrappedVerifiableCredential(vc as OriginalVerifiableCredential).original;
 };
 
-export const getCredentialStatus = (credential: ICredential | VerifiableCredential | ICredentialSummary): CredentialStatusEnum => {
+export const getCredentialStatus = (credential: ICredential | VerifiableCredential | ICredentialSummary, expirationDate: number): CredentialStatusEnum => {
```

Comment:
expirationDate should not be a separate param, this should be in the credential. now you could get a missmatch

Reply:
handled it within the `isExpired` method instead of passing the value again